vspace/src/main/webapp/WEB-INF/views/staff/modules/slides/slide.html

```diff
@@ -66,102 +75,155 @@
         $(".open").css('height', '400px');
     } else {
     	// store image ID selected by the user to replace, onDoubleClick
-    	var imgID = $(e.target).attr('id'); 
-    	if(imgID != ''){
-        	$("#uploadImage").data('value', imgID); // sets image ID value
+    	imgBlockID = $(e.target).attr('id'); 
```

Comment:
where is this variable declared?

Reply:
at the top.As it is needed afterwards so made it global.

Reply:
why does this need to be global? It's doesn't seem to be used.